### PR TITLE
Update chronological-issues-from-video.md

### PR DESCRIPTION
**Pull request by n4n0b1t3**
Content update for video issue page containing a note about the confusion between network ID and chain ID which often leads to unexpected errors and is the source of quite a few requests in the discussions.
I was asked by @cromewar to contribute to this information. Never contributed to an open GitHub project before, hope I am doing this is correct. If not please give me a hint.

### CODE REVIEW

**Comment by cromewar** [comment]
@PatrickAlphaC  Please give it a check this is valuable information, thanks again @n4n0b1t3 !!

**Comment by PatrickAlphaC** [comment]
Woah.... this is super weird. Do you have any docs that show that these two _should_ be different? It could be a weird ganache thing.... 

Also, you'd doing this great! This is exactly how one would contribute. Could you share some other docs that show that ganache is wonnkey?

**Comment by n4n0b1t3** [comment]
Hmm, actually Ganache behaves correctly towards the specification. I even have the suspicion that Ganache has taken two different values on purpose. After all, it is a development tool and so the developers can learn that these are two different values.
I had referred to the following article: https://besu.hyperledger.org/en/stable/Concepts/NetworkID-And-ChainID/
Just now, I took a look at the ETH chain specification: https://openethereum.github.io/Chain-specification.html

ChainID was introduced after the Network ID, the default value for ChainID is equal to Network ID (but can be different). Here is the text from the specification. 
> "chainID" chain identifier, was introduced in [EIP-155](https://eips.ethereum.org/EIPS/eip-155) to prevent replay attacks between the main ETH and ETC chains, which both have a networkID of 1. ChainID is an additional way to tell chains apart. Subsequent to EIP-155, ETH had a chainID of 1, while ETC had a chainID of 61. networkID and chainID are required for the chain to operate in general - e.g. it’s required when signing transactions. **Default value is equal to networkID**.

File: chronological-issues-from-video.md [hunk 1]
```diff
@@ -73,6 +73,23 @@ transaction = SimpleStorage.constructor().buildTransaction(
...
+>>> from web3 import Web3
+>>> w3 = Web3(Web3.HTTPProvider("http://127.0.0.1:8545"))
+>>> w3.eth.chain_id
+1337
+>>> w3.net.version
+'5777'
+>>> 
+```
+
+One possibility is setting the chain_id programmatically.
```
Comment (PatrickAlphaC) [inline chronological-issues-from-video.md:90]:
```suggestion
```

File: chronological-issues-from-video.md [hunk 2]
```diff
@@ -73,6 +73,23 @@ transaction = SimpleStorage.constructor().buildTransaction(
...
+`"chainId": w3.eth.chain_id, `
```
Comment (PatrickAlphaC) [inline chronological-issues-from-video.md:91]:
```suggestion
```

**Review by PatrickAlphaC — APPROVED** [review]

**Comment by PatrickAlphaC** [comment]
Lovely.

Thank you for this!

I didn't know that, learn something new everyday :)

**Comment by PatrickAlphaC** [comment]
Great first contribution!!

**Comment by n4n0b1t3** [comment]
It is fun to work on the project and I learn a lot! Thank you very much, this is a great team.

**Pull request merged**